getting_started.md

```diff
@@ -0,0 +1,104 @@
+
+![Scuttlebutt Logo](assets/hermies-300x300.png)
+
+Ok, you've been hearing about [Scuttlebutt](https://scuttlebutt.nz)  and decided that "Yes, I do want to join an amazing social network with lots of good people that no company can control and also happens to also work offline. 
+
+Here's a quick overview with the basics you need to know.
+
+First off Scuttlebutt is a protocol on which many different kinds of apps can be built. 
+
+As for the social network, there are many clients, just like there are many Twitter clients. It doesn't really matter which one you use. They're all talking on the same network. 
+
+Patchwork is the most polished client and it doesn't require you to know any geeky stuff. The only caveat is that it just wants to run on one computer, and your identity is tied to the files on the computer you install it on. There are ways around this but it's beyond the scope of this.
+
+![screenshot of Patchwork](assets/patchwork.jpg)
+
+
+## Step 0
+If you haven't done so already go to [the Scuttlebutt Home Page](README.md) and watch the video. Then come back. 
+
+## Step 1
+[Download Patchwork](http://dinosaur.is/patchwork-downloader/), and install it like any other app. 
+ 
+## Step 2
+Boot it up and set up your profile. 
+
+![patchwork new profile screenshot](assets/patchwork-first-boot.png)
+
+It doesn't matter what name you put. It can be your real name. It can be a nickname. Like the real world it's entirely possible for multiple people to have the same name. Your friends can give you nicknames too. 
+
+You don't need to upload an image, but I recommend that you do, because it's much easier to figure out who is talking when there's a visual, and the defaults are really generic. Some people use a photo of themselves. Some people use a random avatar they like, just like Twitter.
+
+The text area, like every text area in Patchwork uses [markdown](http://commonmark.org/help/) formatting.
+
+Save that when you're done.
+
+## Step 3 
+
+Get connected.  
+Because Scuttlebutt is a distributed network you need to find some people to follow. This is partially because it's really _really_ boring if you're not listening to anyone and no-one is listening to you. It's also because you get "gossip" from the people you follow. "Gossip" is more than just info about people you're interested in. It is also the name of [a protocol](https://en.wikipedia.org/wiki/Gossip_protocol) that Scuttlebutt uses to share information. 
+
+### Connecting Locally
+Most of us don't happen to be sitting next to someone else on the network. If you _are_ you'll see them listed under "local" on the left. Click their profile to see more about them, and click "Follow" if you feel like following what that person has to say (just like Twitter).  You'll start downloading information from them that they've posted to the network _and_ information their friends have posted. 
```

Comment:
@glowkeeper pointed out that he didn't see the "Local" which means this isn't clear enough about it only being visible IF someone else is running ssb on your local network.